packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -758,6 +854,11 @@ class ModalBottomSheetRoute<T> extends PopupRoute<T> {
   /// Defaults is true.
   final bool enableDrag;
 
+  /// Specifies whether drag handler is displayed when the drag action is enabled.
+  ///
+  /// Defaults is true in material 3 and false in material 2.
+  final bool? hasDragHandle;
```

Comment:
```suggestion
  /// Specifies whether a drag handle is shown when [enableDrag] is true.
  ///
  /// When null (the default), behaves as true when [ThemeData.useMaterial3]
  /// is true, and otherwise behaves as false.
  final bool? hasDragHandle;
```